server/database.sql

```diff
@@ -102,7 +102,8 @@ CREATE TABLE IF NOT EXISTS `user_groups` (
     FOREIGN KEY (`username`) 
     REFERENCES `users` (`username`) 
     ON DELETE CASCADE 
-    ON UPDATE CASCADE) 
+    ON UPDATE CASCADE,
+  UNIQUE `unique_member`(`username`, `group_id`)) 
```

Comment:
Added this to prevent duplicate rows of username-group_id map. Because, it might happen that some user requests to add the same username multiple times in a group.

Reply:
ok